@navikt/core/css/darkside/button.darkside.css

```diff
@@ -17,26 +17,28 @@
   box-shadow: inset 0 0 0 var(--__axc-button-border-width) var(--__axc-button-border-color);
 
   &:focus-visible {
-    outline: 2px solid var(--ax-border-focus);
-    outline-offset: 2px;
```

Comment:
This is part of a new test for focus-markings. I will create a separate PR for all the other components updating this.